Source/WebKit/NetworkProcess/NetworkSession.h

```diff
@@ -206,8 +206,9 @@ class NetworkSession : public WebCore::SWServerDelegate, public CanMakeCheckedPt
     ServiceWorkerFetchTask* navigationPreloaderTaskFromFetchIdentifier(WebCore::FetchIdentifier);
     void removeNavigationPreloaderTask(ServiceWorkerFetchTask&);
 
-    WebCore::SWServer* swServer() { return m_swServer.get(); }
-    WebCore::SWServer& ensureSWServer();
+    WebCore::SWServer* swServer() { return m_swServer.get(); } // FIXME: Rename this to swServerIfExists
```

Comment:
FIXMEs not really related to this PR and I'm not sure I understand why the proposal is better.

Reply:
This is the naming convention we agreed to in 2013: https://lists.webkit.org/pipermail/webkit-dev/2013-June/025056.html

Reply:
I'm aware but I don't think we ever applied / enforced it.

Reply:
This code doesn't pre-date 2013

Reply:
I don't think it matters. The fact of matter is that is the naming convention we had agreed to so that's what we should be using. ensureX / x variant should be phased out.